### ISSUE TITLE
Pickers A11y Pass

### ISSUE THREAD

**Issue by awalia13** [OfficeDev/office-ui-fabric-react#8655]
Tracking all of the issues brought up with the 04/08 accessibility pass on Pickers control
Details on individual bugs can be found on [Sharepoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B4b0ae7d5-d7f3-4ce3-bdf4-0c363fedbdb3%7D&action=default&uid=%7B4B0AE7D5-D7F3-4CE3-BDF4-0C363FEDBDB3%7D&ListItemId=1948&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble)

 - [ ] Bug1:[Accessibility][Programmatic access]: ARIA attributes must conform to valid values
 - [x] Bug2:[Accessibility][Screen Readers]: Edit field under Custom Picker (Document Picker) variant does not have a label.
 - [ ] Bug3:[Accessibility][Screen Readers]: No name is defined for remove button present next to selected colors.
 - [ ] Bug4:[Accessibility][Usable]: Screen reader is not providing information about no suggestion appearing on screen when user provide inappropriate entry in edit field .
 - [ ] Bug5:[Accessibility][Usable]: Focus is not landing on the first selection of the edit field under Tag Picker variant rather going to remove button.
 - [ ] Bug6:[Accessibility][Supporting the Platform]: Remove button present next to colors is not visible when focus goes on them in high contrast mode white and black.
 - [ ] Bug7:[Accessibility][Keyboard Navigation]: Suggested tags text appears as if it’s a link due to its color

**Comment by joschect** [comment]
@awalia13 thanks for submitting this! We'll add it to our backlog and make sure that the right people look into it.

**Comment by suryasu** [comment]
For bug #2, what would we want the aria label to say? Since it's a custom picker, the label should be defined by whoever is creating the custom element right? Labeling it as `Custom Picker` or just `Picker` wouldn't be useful for screen readers

**Comment by micahgodbolt** [comment]
Yeah, we don't provide strings in our controls, but we should ensure that proper strings can be provided. Since the testing is done on our docs pages themselves, we might just need to update the website to properly include that label.

**Comment by dzearing** [comment]
@joschect ping.
